scripts/pull_request_blaster_outer/lockdown_ruby_version.rb

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env ruby
+require 'yaml'
+
+if `grep -E 'git clone.+manageiq.git' bin/setup`.include?('manageiq.git')
+  if File.exist?(File.join(Dir.pwd, 'bin/setup'))
+
+    travis = File.join(Dir.pwd, '.travis.yml')
+    unless File.exist?(travis)
+      puts "#{travis} doesn't exist, skipping"
+      exit 0
+    end
+
+    travis_content = File.read(travis)
+    changed = false
+    changed = true if travis_content.gsub!(/2\.[0-9].[0-9]+/, '2.5.3')
```

Comment:
Maybe we should drop this section since it doesn't do the excludes correctly

Reply:
Note, if we drop this, then this script becomes similar to some of the other scripts so we might want to consolidate them.  Maybe?